docs/installation/network-options.md

```diff
@@ -182,10 +182,21 @@ Single-stack IPv6 clusters (clusters without IPv4) are supported on K3s using th
 --cluster-cidr=2001:cafe:42:0::/56 --service-cidr=2001:cafe:42:1::/112
 ```
 
-
 ## Distributed hybrid or multicloud cluster
 
-A K3s cluster can still be deployed on nodes which use different private networks and are not directly connected (e.g. nodes in different public clouds). To achieve this, K3s sets a mesh of tunnels that become a VPN mesh. These nodes must have have an assigned IP through which they can be reached (e.g. a public IP). The server traffic will use a websocket tunnel and the data-plane traffic will use a wireguard tunnel.
+A K3s cluster can still be deployed on nodes which use different private networks and are not directly connected (e.g. nodes in different public clouds). There are two options to achieve this: wireguard-native and tailscale
```

Comment:
```suggestion
A K3s cluster can be deployed on nodes which do not share a common private network and are not directly connected (e.g. nodes in different public clouds). There are two options to achieve this: the `wireguard-native` flannel backend, and the `tailscale` VPN provider.
```

Reply:
This is not accurate though, because one option is not `wireguard-native`. That is what pod traffic is using, however, this is not what control-planes are using. They use public IP + https to communicate. So, summarizing the solution by saying `wireguard-native` is misleading. I think we should find a better naming, something like: 
```
There are two options to achieve this: the k3s internal multicloud solution, and the integration with tailscale VPN provider.
```
And then describe both